<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="TemTable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hourly Salary</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Worked Hours</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="PermTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Worked Days</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Monthly Salary</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Number of Children</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marital Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Monthly Bonus</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="PermanenEmployee">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Permanent Employee Management</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TemporealEmployee">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Temporal Employee Management</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Permanent Employees</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Temporal Employees</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
